src/Galerkin/quadratures.jl

```diff
@@ -28,6 +28,11 @@ function default_quadrature(Bs::Tuple{Vararg{AbstractBSplineBasis}})
     _quadrature_prod(Val(polynomial_order))
 end
 
+function _gausslegendre_impl(::Val{n}) where {n}
```

Comment:
because this is used in a generated function, it needs to be defined before the `@generated` to avoid world-age issues